package.json

```diff
@@ -28,6 +28,7 @@
     "date-fns": "^2.16.1",
     "dotenv": "^8.2.0",
     "feather-icons-react": "^0.4.3",
+    "knex": "^0.95.6",
```

Comment:
Correct if I'm wrong, but I believe there shouldn't be any reason to import 'knex' into the frontend repo, no? Was this an intentional addition? Just double-checking!

Reply:
Definitely no need for knex. good catch!